### DISCUSSION TITLE
i want change my system  From Ubuntu to windows

### DISCUSSION THREAD

**Discussion by yeshanji** [Chia-Network/chia-blockchain#7693] [Q&A]
Dear All：
     i want change my system  From Ubuntu to windows
     what should i do

**Comment by SlugFiller** [comment]
Backup ~/.chia/mainnet. The two clients use the same codebase, so the data files should be 100% compatible. For extra safety, double and triple check your seed phrase, down to the last spelling error.

For reference, on Ubuntu ~ is something like /home/neo, and in Windows ~ is something like C:\Users\Anderson. Also, in Ubuntu bash "cd ~" would work, but in Windows cmd you'd have to type "%HOMEDRIVE%;cd %HOMEDRIVE%%HOMEPATH%"

Also, presumably you're not formatting any of your plot drives, otherwise you'll have to replot, obviously. But, replotting is just a bit of extra time, so it's no huge loss even if you end up doing it.
